posts/2022-08-04-jakarta-core-profile-22009-beta.adoc

```diff
@@ -0,0 +1,270 @@
+
+* Two new Observable Container State Events have been added; Startup and Shutdown. Applications can listen for these events to be notified when the CDI container is starting up and being shut down.
+
+* A new Handle API is added to make programatic lookup of Beans and Instances easier.
+
+* (Jakarta Annotations 2.1) The `@priority` annotation can now be applied to any program element. This is used by CDI 4.0 to allow Stereotypes to include a Pritory annotation.
+
+* Some previously deprecated CDI APIs have been removed.
+** @New qualifier
+*** Replaced by `@dependent` beans (since CDI 1.1)
+
+** Bean#isNullable()
+*** Not used by the implementation (since CDI 1.1)
+
+** BeanManager#fireEvent()
+*** Use `BeanManager.getEvent()` instead (since CDI 2.0)
+
+** BeanManager#createInjectionTarget(AnnotatedType)
+*** Replaced by `BeanManager#getInjectionTargetFactory(AnnotatedType)` (since CDI 1.1)
+
+** BeforeBeanDiscovery#addAnnotatedType(AnnotatedType)
+*** Replaced by `BeforeBeanDiscovery#addAnnotatedType(AnnotatedType, String)` (since CDI 1.1)
+
+* The Liberty configuration element has been superseded by a new element that applies to all versions of CDI (1.2 and newer).
+
+[source, xml]
+----
+<cdi enableImplicitBeanArchives="false" emptyBeansXmlCDI3Compatibility="true"/>
+----
+
+* The attribute `enableImplicitBeanArchives` is the same as it was in previous versions.
+If set to `true` (default) then archives which do not contain a `beans.xml` file will be treated as Implicit Bean Archives and will be scanned for classes that have Bean Defining Annotations.
+If set to `false` then archives which do not contain a `beans.xml` file will be not be scanned for annotated classes.
+
+* The attribute `emptyBeansXmlCDI3Compatibility` only applies to CDI 4.0.
+If set to `true` then an archive which contains an empty `beans.xml` file will be treated as an Explicit Bean Archive (as it was in CDI 3.0 and earlier)
+
+* If set to `false` (default) then an archive which contains an empty `beans.xml` file will be treated as an Implicit Bean Archive. This is the equivalent to setting `bean-discovery-mode="annotated"`.
+
+[source, xml]
+----
+<featureManager>
+  <feature>cdi-4.0</feature>
+</featureManager>
+
+<cdi enableImplicitBeanArchives="false" emptyBeansXmlCDI3Compatibility="true"/>
+----
+
+[#expression]
+== Jakarta EE 10 Expression Language 5.0
+
+The `expressionLanguage-5.0` feature is an implementation of the Expression Language 5.0 specification for Jakarta EE 10. The Expression Language 5.0 includes a number of new features and specification clarifications.
+
+To learn about the changes between the Expression Language 4.0 and Expression Language 5.0, reference the  link:https://jakarta.ee/specifications/expression-language/5.0/jakarta-expression-language-spec-5.0.html#changes-between-5-0-and-4-0[Expression Language 5.0 spec documentation].
```

Comment:
Remove this paragraph - don't tell the reader to "go read the spec". Provide them with the basic info they need here, then link to the relevant topics in the OL docs. In a beta, the docs might not yet be written so, in the case, if necessary link to the spec for more info at the end of the section.